src/Nncase.CodeGen/Nncase.CodeGen.csproj

```diff
@@ -8,7 +8,6 @@
 
   <ItemGroup>
     <PackageReference Include="Extension.Mathematics" />
-    <PackageReference Include="Razor.Templating.Core" />
```

Comment:
Template engine requires this package?

Reply:
no, this package will be used in Nncase.Modules.CPU

Reply:
OK.